gapic/src/main/com/google/gapid/widgets/ImagePanel.java

```diff
@@ -1056,9 +1057,11 @@ public void init(Renderer renderer) {
     public void update(Renderer renderer, SceneData newData) {
       // Release textures that are no longer in data.
       Set<Image> newSet = Sets.newHashSet(newData.images);
-      for (Map.Entry<Image, Texture> entry : imageToTexture.entrySet()) {
+      for (Iterator<Map.Entry<Image, Texture>> it = imageToTexture.entrySet().iterator(); it.hasNext(); ) {
```

Comment:
While this is marginally faster, I found the implementation in #1346 significantly easier to read.